install_config/install/docker_registry.adoc

```diff
@@ -173,20 +173,52 @@ $ oc logs docker-registry-1-da73t
 
 == File Storage
 
-Tag and image metadata is stored in OpenShift, but the registry owns layer and
-signature data. The registry stores data in a volume that is mounted into the
-registry container at *_/registry_*. You can
+Tag and image metadata is stored in OpenShift, but the registry stores layer and
+signature data in a volume that is mounted into the registry container at
+*_/registry_*.
+
+To see a registry's contents, ssh into the pod, then list the contents:
+
+. List the current pods to find the desired pod name:
```

Comment:
s/desired pod name/pod name of your Docker registry/

^ Might as well be specific since we're talking about the registry pod here.